core/common/src/test/java/alluxio/CommonTestUtils.java

```diff
@@ -121,7 +122,11 @@ public static void waitFor(String description, Function<Void, Boolean> condition
       }
       field.setAccessible(true);
       T instance = createBaseObject(clazz);
-      field.set(instance, getValuesForFieldType(field.getType()).get(1));
+      try {
```

Comment:
The exception is still thrown, but now it isn't checked. Throwing a checked exception here places an unnecessary burden on callers, which never care about the specific type of exception thrown.